docs/logs/analysis-tab.asciidoc

```diff
@@ -30,14 +28,36 @@ Once you have selected the time range, click *Create ML job* to create the machi
 Now you can start detecting anomalies in your logs.
 
 [float]
+[[logs-analysis-page-view-log-entries]]
 === View log entries
 
-Once the machine learning job has been created, the *Analysis* page shows the log entries chart, an overall anomalies chart, and a collapsible entry for the anomalies in each partition.
+Once the machine learning job has been created, the *Analysis* page shows:
+* the log entries chart
+* an overall anomalies chart
+* the anomalies in each partition.
 
 The time range over which the logs are analyzed is fixed at the time range you selected when you created the machine learning job.
-But you can use the time filter at the top of the *Analysis* page to restrict the period for which the results are shown.
+But you can use the time filter at the top of the *Analysis* page to restrict the time range for which the results are shown.
+
+[float]
+[[logs-analysis-page-change-time]]
+=== Changing the time range
+
+Use the time filter to select the time range for the results shown in the anomaly charts.
+
+To quickly select some popular time range options, click the clock dropdown image:logs/images/time-filter-clock.png[]. In this popup you can choose from:
+
+* *Quick select* to choose a recent time range, and use the back and forward arrows to move through the time ranges
+* *Commonly used* to choose a time range from some commonly used options such as *Last 15 minutes*, *Today*, or *Week to date*
+* *Refresh every* to specify an auto-refresh rate
+* *Stop* to stop auto-refresh (enabled by default for logs anomaly charts)
+
+NOTE: When you stop auto-refresh from within this dialog, the clock dropdown changes to a calendar image:logs/images/time-filter-calendar.png[].
+
+For complete control over the start and end times, click the start time or end time shown in the bar beside the calendar or clock dropdown. In this popup, you can choose from the *Absolute*, *Relative* or *Now* tabs, then specify the required options.
```

Comment:
Do we really need to explain how to use the time filter here?

Reply:
Removed in https://github.com/elastic/kibana/pull/50832. Open to feedback.

Reply:
@bmorelli25 I agree.  I think the time filter is pretty intuitive and we don't need to explain it everywhere.